app/controllers/ConfigurationController.scala

```diff
@@ -39,8 +37,9 @@ class ConfigurationController @Inject()(val messagesApi: MessagesApi) extends Co
 
   def readDataSet(dataSetName: String) = UserAwareAction.async { implicit request =>
     request.identity.toFox.flatMap { user =>
-      UserService.findOneById(user.id, useCache = false)
-      .flatMap(_.dataSetConfigurations.get(dataSetName))
+      for {
```

Comment:
This seems to be more than a refactoring. If I understand it correctly, this used to be cached in the `UserCache`, not sure if it was used from there.

And for my understanding: `findOneForUserAndDataset` comes from slick now, right?

Reply:
Correct observation, I did not consider that this is now bypassing the Cache. However, I believe that it is not problematic. This configuration is requested via a separate route (not part of the user json) and that route is requested far less often. Also, the UserCache’s cache invalidation should provide that there is not actually a change in behavior (albeit maybe a tiny one in performance)

Reply:
Fine for me :+1: